sherpa/astro/datastack/ds.py

```diff
@@ -28,6 +27,10 @@
 
 logger = config_logger(__name__)
 
+try:
+    import stk
+except:
+    logger.warning("could not import stk library. CIAO stack files and syntax will be disabled")
```

Comment:
That is exactly what I had in mind. Yes, we could improve the documentation (as a matter of fact I had a comment that explained that but apparently I dropped it after changing the `try..catch` block. If the stack library is not present and you use the stack syntax (e.g. `@filename`) you'll get a file not found error, which paired with the warning should be reasonable enough.